### ISSUE TITLE
CLI: update command returns wrong version (Checks against archived repository)

### ISSUE THREAD

**Issue by dallyger** [bitwarden/clients#3062]
### Steps To Reproduce

- Download the latest CLI release
- Install `bw` binary
- Change server to self-hosted instance
- Run `bw update`

### Expected Result

```
❯ bw update
No update available.
```

### Actual Result

```
❯ bw update
A new version is available: v1.22.1
- New --hostname option for the bw serve command to securely bind your API web server to a host
- Bug fixes

You can download this update at https://github.com/bitwarden/cli/releases/download/v1.22.1/bw-linux-1.22.1.zip

If you installed this CLI through a package manager you should probably update using its update command instead.
```

### Operating System

Linux

### Operating System Version

Pop!_OS 22.04 LTS - 5.17.5-76051705-generic

### Shell

Zsh

### Build Version

2022.6.1

**Comment by dbosompem** [comment]
@dallyger can you try again and confirm if issue still persists?

**Comment by kpiris** [comment]
I’m not @dallyger, but I can confirm it's not fixed in version 2022.8.0:

```
$ bw -v
2022.8.0

$ bw update
A new version is available: v1.22.1
- New --hostname option for the bw serve command to securely bind your API web server to a host
- Bug fixes

You can download this update at https://github.com/bitwarden/cli/releases/download/v1.22.1/bw-linux-1.22.1.zip

If you installed this CLI through a package manager you should probably update using its update command instead.
```

**Comment by dallyger** [comment]
@dbosompem can confirm that the issue still persists with the latest versions. I've tested it right now, and I get the exact same output as @kpiris.

**Comment by dbosompem** [comment]
Alright, thanks for the confirmation, I will pass this on to the team to take a look.

**Comment by modem7** [comment]
Can confirm this is still occuring with 2022.10.0

**Comment by kpiris** [comment]
This issue is not fixed on the version just released:

```
$ bw -v
2022.11.0

$ bw update
A new version is available: v1.22.1
- New --hostname option for the bw serve command to securely bind your API web server to a host
- Bug fixes

You can download this update at https://github.com/bitwarden/cli/releases/download/v1.22.1/bw-linux-1.22.1.zip

If you installed this CLI through a package manager you should probably update using its update command instead.
```

**Comment by djsmith85** [comment]
@kpiris: This is expected. The linked PR only got merged a couple of hours ago. The linked changes will be part of the next major cli-release.
